message_to_tf/src/message_to_tf.cpp

```diff
@@ -34,7 +35,7 @@ ros::Publisher g_euler_publisher;
 
 void addTransform(std::vector<geometry_msgs::TransformStamped>& transforms, const tf::StampedTransform& tf)
 {
-  transforms.resize(transforms.size()+1);
+  transforms.push_back(geometry_msgs::TransformStamped());
```

Comment:
Slightly cleaner syntax for initializing a new TransformStamped on the back of the vector.